benchmark/httpapi.py

```diff
@@ -114,10 +137,11 @@ def post(self, request):
         request.setHeader(b'content-type', b'application/json')
```

Comment:
I think this now has to go after the error block (at least, I know there are HTTP server implementations that don't let you call setHeader twice for the same header, and _bad_request now also tries to set content-type.

If I'm wrong, ignore me.